cli.py

```diff
@@ -66,11 +72,8 @@ def attr_upd():
             print(appliance_manager.aircons)
             print(appliance_manager.washer_dryers)
             print(appliance_manager.ovens)
-            return
-
-        if not args.said:
-            logger.error("No appliance specified")
-            return
+            print(appliance_manager.beer_fridges)
+            # return
```

Comment:
```suggestion
            return
```
the `--list` should only show the list output and terminate